modules/bbp-electrophysiology/src/main/resources/schemas/bbp/electrophysiology/trace/v0.1.0.json

```diff
@@ -24,64 +24,81 @@
       "and":[
         {
           "node":"bbpentsh:EntityShape"
-        },{
+        },
+        {
           "property" : [ {
             "path": "nsg:projectName",
             "name": "Project name",
             "description": "Name of the project that the trace was recorded for",
             "datatype": "xsd:string",
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:sweep",
             "name": "Sweep",
             "description": "Sweep number of trace",
             "datatype": "xsd:integer",
             "minCount": 1,
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:repetition",
             "name": "Repetition",
             "description": "Repetition number of trace",
             "datatype": "xsd:integer",
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:channel",
             "name": "Channel",
             "description": "Channel number of trace",
             "datatype": "xsd:integer",
             "minCount": 1,
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:originalFileName",
             "name": "Original filename",
             "description": "Original filename of binary",
             "datatype": "xsd:string",
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:retrievalDate",
```

Comment:
What is the intent of this property ?

Reply:
The intention was to store the link between metadata and binary file in cases where the binary file is not simultaneously attached to the payload. As I understood it, this would then be overwritten by the platform's field "originalFileName" as soon as a binary is attached.

Reply:
It would be overwritten if nxv:originalFileName is used. Here nsg: originalFileName is used.